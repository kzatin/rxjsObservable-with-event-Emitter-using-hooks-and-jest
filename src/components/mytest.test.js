import React from 'react';
import "@testing-library/jest-dom"
import renderer from 'react-test-renderer'
import Temprature from './temperature'
import AirPressure from './airPressure'
import Humidity from './humidity'


//snapshot testing

it("temperature matches snapshot",()=>{
   const tree = renderer.create(
        <Temprature/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
})

it("pressure matches snapshot",()=>{
    const tree = renderer.create(
         <AirPressure/>
     ).toJSON();
 
     expect(tree).toMatchSnapshot();
 })

 it("humidity matches snapshot",()=>{
    const tree = renderer.create(
         <Humidity/>
     ).toJSON();
 
     expect(tree).toMatchSnapshot();
 })

 
