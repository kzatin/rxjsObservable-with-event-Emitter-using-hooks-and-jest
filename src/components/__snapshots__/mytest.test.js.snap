// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`humidity matches snapshot 1`] = `
<div>
  The Humidity is 
  .
</div>
`;

exports[`matches snapshot 1`] = `
<div>
  The Temprature is 
  .
</div>
`;

exports[`pressure matches snapshot 1`] = `
<div>
  The Air Pressure is 
  5
  .
</div>
`;

exports[`temperature matches snapshot 1`] = `
<div>
  The Temprature is 
  .
</div>
`;

exports[`temperature matches snapshot 2`] = `
<div>
  The Temprature is 
  5
  .
</div>
`;

exports[`temperature matches snapshot 3`] = `
<div>
  The Temprature is 
  .
</div>
`;
